src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/BatchController.java

```diff
@@ -22,7 +24,7 @@
 import static uk.gov.justice.digital.hmpps.keyworker.batch.PrisonStatsRoute.DIRECT_PRISON_STATS;
 import static uk.gov.justice.digital.hmpps.keyworker.batch.UpdateStatusRoute.DIRECT_UPDATE_STATUS;
 
-@Api(tags = {"batch"}, hidden = true)
+@Tag(name = "batch")
```

Comment:
Any reason for removing the "hidden" - I know this should never be called by users, just cron jobs

Reply:
Tag doesnt have a hidden option, all the end points are hidden though, they also have security settings that i assume do not work (you would need to log in to see them).  My assumption is a lof of the api references are cut and paste code, given the differences and un-used things

Reply:
yeah, ok